### ISSUE TITLE
defaultSeccompProfile = "/etc/sysconfig/manhattan.json"

### ISSUE THREAD

**Issue by runcom** [GrantSeltzer/Manhattan#17]
is that value correct? `manhattan.json`?

Also, what `default.json` does in this repository?

**Comment by grantseltzer** [comment]
@runcom The name is arbitrary. The install option should move default.json to that location. If you have no objection to that idea I'll add it to the makefile and write about it in the README.md

**Comment by runcom** [comment]
Is the default coming from Docker? If that's the case I'd prefer docker installation to take care of installing it or otherwise it won't always be in sync

**Comment by grantseltzer** [comment]
Yes it is, and sure. Do you know if when Docker is installed if a default seccomp profile is placed somewhere?

**Comment by runcom** [comment]
It isn't (I'm maintaining  the package in Fedora)

We shall leave this opened to track this

**Comment by grantseltzer** [comment]
For now should I do it the way I originally suggested?

**Comment by runcom** [comment]
yes, we'll remove the Makefile target after we fix docker installation

**Comment by grantseltzer** [comment]
:+1:

**Comment by grantseltzer** [comment]
Leaving this open for now

**Comment by rhatdan** [comment]
Yes it would be out of date.  Having it rely on docker is not great either, especially where we would like to support pure OCI environments.

**Comment by runcom** [comment]
> Yes it would be out of date. Having it rely on docker is not great either, especially where we would like to support pure OCI environments.

right, my proposal is then to have `ocitools` generate a default profile /cc @mrunalp

**Comment by rhatdan** [comment]
Ok that would be better.
